library/src/main/java/com/github/lukaspili/reactivebilling/observable/GetPurchasesObservable.java

```diff
@@ -33,7 +36,10 @@ protected void onBillingServiceReady(BillingService billingService, Observer<? s
         try {
             observer.onNext(billingService.getPurchases(purchaseType, continuationToken));
             observer.onCompleted();
-        } catch (RemoteException e) {
+        } catch (RemoteException | JSONException e) {
+            if(e instanceof JSONException) {
```

Comment:
IMHO catch clauses should only be collapsed if they're identical.  If you have to `instanceof` check in here, they should be separate clauses.  That being said, is there a reason not to log all errors here?